docs/sources/tempo/operations/cross_tenant_query.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cross-tenant query federation
+menuTitle: Cross-tenant query
+description: Cross-tenant query federation
+weight: 70
+aliases:
+- /docs/tempo/operations/cross-tenant-query
+---
+
+
+# Cross-tenant query federation
+
+{{% admonition type=note" %}}
+You need to enable `multitenancy_enabled: true` in the cluster for multi-tenant querying to work.
+see [enable multi-tenancy]({{< relref "./multitenancy" >}}) for more details and implications of `multitenancy_enabled: true`.
+{{% /admonition %}}
+
+Tempo supports multi-tenant queries. where users can send list of tenants  multiple tenants.
+
+The tenant IDs involved need to be specified separated by a '|' character in the 'X-Scope-OrgID' header.
+
+cross-tenant query is enabled by default, and can be controlled using `multi_tenant_queries_enabled` config.
+
+```yaml
+query_frontend:
+   multi_tenant_queries_enabled: true
+```
+
+### Use cross-tenant query federation
+
+To submit a query across all tenants that your access policy has access rights to, you need to configure tempo datasource.
+
+Update Tempo datasource to send `X-Scope-OrgID` header with values of tenants separated by `|` e.g. `test|test1`, and query the tempo like you already do.
+
+<p align="center"><img src="../header_ds.png" alt="X-Scope-OrgID Headers in Datasource"></p>
+
+If you are provisioning tempo datasource via Grafana Provisioning, you can configure `x-scope-orgid` header like this:
+
+```yaml
+  jsonData:
+    httpHeaderName1: 'x-scope-orgid'
+  secureJsonData:
+    httpHeaderValue1: 'test|test1'
+```
+
+> NOTE: for streaming search, header `x-scope-orgid` needs to be lowercase.
```

Comment:
Changing this to use the admonition shortcode. 

```suggestion
{{% admonition type="note" %}}
For streaming search, the header `x-scope-orgid` needs to be lowercase.
{{% /admonition %}}

```

Reply:
doing streaming in follow up PR, will drop this.